src/frontend/admin/src/layouts/dashboard/nav/DashboardNav.tsx

```diff
@@ -11,6 +12,16 @@ interface Props {
 
 export function DashboardNav(props: Props) {
   const theme = useTheme();
+
+  useEffect(() => {
+    const body = document.getElementsByTagName("body").item(0);
+    if (body) {
+      setTimeout(() => {
+        body.style.overflow = "auto";
+      });
+    }
+  }, [props.open]);
+
```

Comment:
Hmmm... the real question is why body CSS overflow property is set to `hidden` ? Then if this is enforce by MUI why we do not fix this behavior through `globals.scss` ?